routing.py

```diff
@@ -0,0 +1,40 @@
+import urllib.parse as urlparse
+from typing import Tuple
+
+from domain.metric_type import MetricType
+from domain.types import AgentID
+
+
+def encode_matrix_path(metric: MetricType) -> str:
+    if metric == MetricType.PACKET_LOSS:
+        return "/packet_loss"
+    if metric == MetricType.JITTER:
+        return "/jitter"
+    if metric == MetricType.LATENCY:
+        return "/latency"
+    return "/"
```

Comment:
We need to log an error here, because this should never happen.

Reply:
Done.